tools/xmlschema.rb

```diff
@@ -130,8 +135,7 @@ def printIncludeRef(_file, _spaces, _inc)
 
 #################################################
 def printInclude(_file, _spaces, _attr)
-  loc = "http://sdformat.org/schemas/"
-  loc += _attr.attributes['filename'].sub("\.sdf","\.xsd")
+  loc = _attr.attributes['filename'].sub("\.sdf","\.xsd")
```

Comment:
I should have been more clear in the issue description in that we do still want to be able to have xsd files that can be stored on sdformat.org/schemas. At a glance, this looks like it will embed the absolute paths of the local machine in to the generated schemas. That's useful for tests, but I think we do still want this script to generate xsd that can be uploaded to the website.

I wonder if instead of using `<xsd:include>`, we just paste the contents and generate one xsd file that can be used for both testing and for the website. This might also solve the problem I mentioned in https://github.com/osrf/sdformat/pull/535/files#r609021282

Reply:
How's this? aef3ab859c20cdc955c10fe48a11343966f28d27